### DISCUSSION TITLE
How do you setup Windows VHD Plugin?

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1358] [Q&A] [answered]
I am having trouble setting up the Windows VHD plugin. I am on Linux Mint and want to achieve this by only using open source software.

I downloaded Ventoy GUI and flashed my USB stick for Ventoy, I then placed the Windows 10 VHD plugin onto the stick and then I created a VHD file using VirtualBox. The Virtual Machine VHD file is a fresh Windows 10 installation. I copied the VHD file I made using VirtualBox onto the USB stick for Ventoy but I am unable to boot up the USB stick.

Any help and advice is most appreciated

**Comment by steve6375** [comment]
First, the Ventoy large partition should be NTFS.
Secondly, you should not try to boot a fresh install because on first boot it will attempt to install boot files. So always boot from the vhd drive several times after a fresh install, before copying the vhd file to the Ventoy usb Drive.
See Ventoy documentation pages.

**Reply by trymeouteh** [reply]
UPDATE: I placed the plugin file into the Ventoy/ventoy folder and placed the VHD file into the Ventoy/BOOTIMG folder. When boot up Ventoy I see my Windows VHD as an option to boot into and when I boot into it I get a black screen for a minute and then my computer crashes. How do I get pass this one?

**Comment by steve6375** [comment, marked as answer]
Read the Ventoy documentation page on windows vhd files.

**Reply by steve6375** [reply]
https://ventoy.net/en/plugin_vhdboot.html
The vhd file should be put in the root or a different folder ideally, not the \ventoy folder or subfolder.
Need more details about the VHD. Size of VHD? Type (Fixed/Dynamic)? Legacy boot or MBR boot? Partitions on VHD? etc.

**Reply by trymeouteh** [reply]
VHD Size = 10GB
Type = Dynamic (I set the max size to about 15GB)
MBR Boot on Ventoy USB
VHD has not additional partitions, just installed Windows 10 to the VHD normally without any other configuration.

**Reply by steve6375** [reply]
Ok, I don't know what the problem is. Try booting on some different systems.
If that fails try making a fixed vhd.

**Reply by trymeouteh** [reply]
I solved it. When I moved the VHD file in the Ventoy partition (not inside a folder) all does work.

**Comment by thomaslong1** [comment]
Read this:
https://www.windowscentral.com/how-create-and-set-vhdx-or-vhd-windows-10
I hope you get some information here about your problem.
